transformations/transformations/pool_allocator.py

```diff
@@ -13,7 +13,8 @@
     SymbolAttributes, BasicType, DerivedType, Quotient, IntLiteral, LogicLiteral,
     Variable, Array, Sum, Literal, Product, InlineCall, Comparison, RangeIndex, Cast,
     Intrinsic, Assignment, Conditional, CallStatement, Import, Allocation, Deallocation, is_dimension_constant,
-    Loop, Pragma, SubroutineItem, FindInlineCalls, Interface, ProcedureSymbol, LogicalNot, dataflow_analysis_attached
+    Loop, Pragma, SubroutineItem, FindInlineCalls, Interface, ProcedureSymbol, LogicalNot, dataflow_analysis_attached,
+    expression, parse_fparser_expression, Scope
```

Comment:
I'd prefer to either import the relevant expressions directly (e.g. `RangeIndex`) or to use the elsewhere commonly used pattern `import loki.expression.symbols as sym`